### DISCUSSION TITLE
Crash when changing empty route group color

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#354] [Q&A] [answered]
verison B13:

After creating a new group for routes and then directly trying to change the color of the group, Cruiser crashes.
![Screenshot_20231028-110023_Microsoft Launcher](https://github.com/devemux86/cruiser/assets/121371629/72f1eb42-163d-4ab4-9aba-7dd00b335be6)

It also crashes when trying to change the color of an already existing group.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

These are _empty_ groups, they should not have menus for color / visible of content.
The group menus exist just for convenience to update the content of _filled_ groups.
As these options are stored per route / track.

It should work properly in the latest:
- https://github.com/devemux86/cruiser/discussions/329

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
